build-storybook.log

```diff
@@ -0,0 +1,5314 @@
+info @storybook/react v6.5.12
```

Comment:
Did you mean to commit this file?

Reply:
I'm assuming it was added by accident.

Reply:
Yeah didnot add it intentionally let me check